packages/core/types/@chainlink/index.d.ts

```diff
@@ -130,6 +131,19 @@ declare module '@chainlink/types' {
     rpcPort?: number
   }
 
+  export type EnvDefaults = { [name: string]: string }
+
+  /**
+   * To add a default override for an env var not present here,
+   * ensure that the adapter context from `expose` is passed
+   * to the `getEnv` call(s) for the variable in question. See
+   * `WS_ENABLED` for example usage.
+   */
```

Comment:
@justinkaseman Here's a note I added about how to add the default overrides. I'll add a similar note to the `getEnv` description as well.

Reply:
Done.